src/options/default-node-options.ts

```diff
@@ -105,7 +105,22 @@ export const defaultNodeOption: RenderOption = {
 
     ['reference']:(node: Node, next: Next) => {
         if (node.attrs.type === 'asset') {
-            return `<img${node.attrs.style ? ` style="${node.attrs.style}"` : ``}${node.attrs['class-name'] ? ` class="${sanitizeHTML(node.attrs['class-name'])}"` : ``}${node.attrs.id ? ` id="${node.attrs.id}"` : ``} src="${sanitizeHTML(node.attrs['asset-link'])}" />`
+            const src = node.attrs['asset-link'];
+            const alt = node.attrs?.['redactor-attributes']?.['alt'];
+            const link = node.attrs.link;
+            const target = node.attrs.target || "";
+            const caption = node.attrs?.['redactor-attributes']?.['asset-caption'] || node.attrs?.['asset-caption'] || "";
+            const style = node.attrs.style;
+            const asset_uid= node.attrs['asset-uid'];
```

Comment:
@nadeem-cs [Minor] we can use destructuring to accessing the properties of node.attrs. like below
const { 
    'asset-link': src, 
    'redactor-attributes': { alt, 'asset-caption': redactorCaption } = {}, 
    link, 
    target = "", 
    'asset-caption': assetCaption = "", 
    style, 
    'asset-uid': assetUid 
} = node.attrs;

const caption = redactorCaption || assetCaption;